installation/resources/compass-overrides-local.yaml

```diff
@@ -207,6 +209,13 @@ global:
     subscription:
       oauthSecret:
         manage: true
+  destinationFetcher:
+    request:
+      skipSSLValidation: true
+      oauthTokenPath: "/cert/token"
+    server:
+      tenantDestinationsEndpoint: "/destination-configuration/v1/subaccountDestinations"
+      sensitiveDataEndpoint: "/destination-configuration/v1/destinations"
```

Comment:
Can you somehow (using env vars) reuse these overrides in the ext svc mock main.go as well otherwise we have to double maintain them:

![image](https://user-images.githubusercontent.com/15074116/187373318-3df61452-6ac5-41aa-9fba-fd40594c2bda.png)

Reply:
This will be added with the e2e tests